dynamallow/model.py

```diff
@@ -141,15 +148,15 @@ def say_hello(self):
                 ))
     """
 
-    def __init__(self, raw=None, **kwargs):
+    def __init__(self, **raw):
```

Comment:
`raw` is meant to represent the raw document, as stored in Dynamo -- which may NOT have all of the same fields defined as our model does (remember the schema can evolve and old records do not update).

By changing this to only be `**raw` we're now losing the ability to keep a copy of the raw object.  I'm actually surprised this didn't break any tests -- we should definitely write some to make sure in the future :)

How does Schematics deal with getting data that isn't defined?